### ISSUE TITLE
Plugin is not functional

### ISSUE THREAD

**Issue by Sibiriak** [MSzturc/obsidian-better-pdf-plugin#46]
Version 1.4.0
Obsidian v.1.3.3

Plugin is disabled by Obsidian, with attempt to activate it I get the message  "Better PDF plugin is no longer functional. We recommend unistallling it".

A lot of my notes use your plugin. Is it possible to fix it? or we've lost the plugin forever

**Comment by senseofeternity** [comment]
Yes, I've also just downgraded to the older Obsidian because of it

**Comment by joleaf** [comment]
Push, same issue.

**Comment by Melancan** [comment]
Hmm, so is the plugin gone for good or can we expect any solution? I have a lot of embedded notes via this plugin, so it would be nice if anyone knows some workaround besides downgrading Obsidian, if possible.

**Comment by joleaf** [comment]
@MSzturc , any statement on this? It is very important :)

**Comment by RaduPrusan** [comment]
Does anybody know what happened with this plugin?

**Comment by jorosy** [comment]
@Melancan 
> Hmm, so is the plugin gone for good or can we expect any solution? I have a lot of embedded notes via this plugin, so it would be nice if anyone knows some workaround besides downgrading Obsidian, if possible.

Hi there, see if "slide note" plugin works for you. I am in the same situation as you. 
However, The slide note plugin use different syntax tho. Maybe you can convert those note syntax in text editor in batch. 
I have also post a feature request to the slide note author to support Better PDF plugin. Let's hope this get implemented soon. I really need this to work for my study notes.

**Comment by Melancan** [comment]
> @Melancan
> 
> > Hmm, so is the plugin gone for good or can we expect any solution? I have a lot of embedded notes via this plugin, so it would be nice if anyone knows some workaround besides downgrading Obsidian, if possible.
> 
> Hi there, see if "slide note" plugin works for you. I am in the same situation as you. However, The slide note plugin use different syntax tho. Maybe you can convert those note syntax in text editor in batch. I have also post a feature request to the slide note author to support Better PDF plugin. Let's hope this get implemented soon. I really need this to work for my study notes.

@jorosy 

Thanks for the suggestion. I see that "Slide note" works nicely to scale down the pdfs, which is convenient when they are embedded. However ,"Better-pdf" also positioned the pages one next to the other along the width of the screen, making everything even more compact. So I hope the plugin still gets fixed.

**Comment by joleaf** [comment]
# Better PDF 2
Hi there!
I created [Better PDF 2](https://github.com/joleaf/obsidian-better-pdf-2-plugin) (a fork of this repo, but updated dependencies), and the plugin works again ;).
So feel free to install this plugin manually (download the latest [release](https://github.com/joleaf/obsidian-better-pdf-2-plugin/releases) (2.x.x)).
If some of you may test this new version and confirm that all works well?
I ~~will~~ create**d** a [pull request](https://github.com/obsidianmd/obsidian-releases/pull/2048) to include the Better PDF 2 Plugin in the official community plugin repo.

> PS: yaml formatting works now for the parameters in Version 2 ;)

**Comment by Melancan** [comment]
@joleaf 

Great, thanks a lot. :) It works for me, I only had to change the scaling parameter a bit to get the same size of the pdfs as I had before.

**Comment by jorosy** [comment]
@joleaf 

Thanks for the great work, really appreciate it. :)
some issues I found. 

* The default Obsidian PDF View won't load any PDF while the plugin loaded.
* Images inside PDF won't render in this plugin, it seems happen to all PDF. Feel free to download the[ sample PDF.](https://www.dropbox.com/s/sei1jscq269l58h/%E6%97%A5%E7%AB%8B%E6%B0%B8%E5%A4%A7%E9%9B%BB%E6%A2%AF_UAS6%E5%88%A5%E5%A2%85%E6%A2%AF%E5%9E%8B%E9%8C%84_Catalog-No.-YT-EL-0170-1.pdf?dl=0).

![2023-06-15_22h26_25](https://github.com/MSzturc/obsidian-better-pdf-plugin/assets/5779706/6eda8732-eae9-4064-87f0-f320d88771b9)

**Comment by joleaf** [comment]
Related to the PR for the Better PDF 2 Plugin: https://github.com/obsidianmd/obsidian-releases/pull/2048#issuecomment-1594969947
> As the original "Better PDF" plugin does not have any license, we can't accept this submission. Please see our [Developer Policies](https://docs.obsidian.md/Developer+policies)
> 
> Sorry about that!

-> @MSzturc can you add a licence to this repo so we can maintain the follow-up better pdf 2 plugin? As an alternative, we can update your project with the required fixes so that the plugin works again. Either way, we need your help.

**Comment by RaduPrusan** [comment]
@joleaf

Better PDF 2.0.3 works well.
Great work. Thanks!

**Comment by Melancan** [comment]
> @joleaf
> 
> Better PDF 2.0.3 works well. Great work. Thanks!

Can we download the 2.0.3 version? I can't seem to find it anywhere.

**Comment by joleaf** [comment]
> > @joleaf
> > Better PDF 2.0.3 works well. Great work. Thanks!
> 
> Can we download the 2.0.3 version? I can't seem to find it anywhere.

https://github.com/joleaf/obsidian-better-pdf-2-plugin/releases/tag/2.0.3

**Comment by Melancan** [comment]
Oh right, I forgot that there's a different page for the better-pdf-2 version. Thanks a lot, it works well now. :)
